v2/audiences_test.go

```diff
@@ -67,26 +67,12 @@ func TestGetAudience(t *testing.T) {
 		}))
 	defer server.Close()
 
-	t.Run("Get Audience", func(t *testing.T) {
+	t.Run("Get Audience Happy Path", func(t *testing.T) {
 		client := courier.CreateClient("key", &server.URL)
 		response, err := client.GetAudience(context.Background(), "123456789")
 		assert.Nil(t, err)
 		assert.Equal(t, expectedResponseID, response.Id)
 	})
-}
-
-func TestAudienceError(t *testing.T) {
```

Comment:
why did we remove this?

Reply:
I moved the test run in to earlier block